File: java/src/test/java/api/jdy/FormDemoTest.java
package api.jdy;

import java.util.Map;

import static constants.HttpConstant.APP_ID;
import static constants.HttpConstant.ENTRY_ID;

public class FormDemoTest {

    public static void main(String[] args) throws Exception {
        FormDemo formDemo = new FormDemo();
        // 表单字段查询接口
        formWidgets(formDemo);
    }

    private static void formWidgets(FormDemo formDemo) throws Exception {
        Map<String, Object> result = formDemo.formWidgets(APP_ID, ENTRY_ID);
        System.out.println("formWidgets result \n" + result);
    }

}

File: python/test/api/jdy/form_demo_test.py
import src.api.jdy.form_demo as form_demo
from src.constants.http_constant import HttpConstant




# 测试 表单字段查询接口
def formWidgets():
    result = form_demo.formWidgets(HttpConstant.APP_ID,HttpConstant.ENTRY_ID)
    print('formWidgets result:', result)

if __name__ == '__main__':
    formWidgets()


